test/behavior/typename.zig

```diff
@@ -1,5 +1,5 @@
+//
```

Comment:
stray line

Reply:
needed for source sensitive test

Reply:
not sure what you're talking about on this one

Reply:
I think this refers to the fact that these files source-location sensitive, as some of these type names contains source lines and columns in the names.